default.nix

```diff
@@ -90,7 +90,8 @@ let
   buildPackage =
     src:
     attrs@
-    { packageLock ? null
+    { name ? "build-npm-package"
```

Comment:
How about we default to whatever is in the lockfile? But 👍 nonetheless

Reply:
Yeah that happens in a later commit, see ee32bd5e1840214601c7cd206d336aee28919df0